### PR TITLE
Adjust database tests with latest changes 

### PR DESCRIPTION
**Pull request by Manuri**
## Purpose
$Subject

### CODE REVIEW

File: stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/Constants.java [hunk 1]
```diff
@@ -197,8 +197,9 @@
     public static final String SQL_FILE_DB_POSTFIX = "_FILE";
     public static final String SQL_MEMORY_DB_POSTFIX = "_MEMORY";
 
-    public static final String SQL_CONNECTOR_ERROR = "error";
-    public static final String SQL_EXCEPTION_OCCURED = "SQL Exception Occurred";
+    public static final String DATABASE_ERROR_RECORD_NAME = "DatabaseError";
+    public static final String DATABASE_ERROR_MESSAGE = "SQL Exception Occurred";
```
Comment (wggihan) [inline stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/Constants.java]: I think we need to change the wording. Better use the word error than the exception. WDYT?

File: stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java [hunk 2]
```diff
@@ -1114,16 +1111,13 @@ public static String getString(Struct udt) throws SQLException {
...
-        StructureTypeInfo errorStructInfo = sqlPackageInfo.getStructInfo(Constants.SQL_CONNECTOR_ERROR);
-        BMap<String, BValue> sqlConnectorError = new BMap<>(errorStructInfo.getType());
-        if (throwable.getMessage() == null) {
-            sqlConnectorError.put(ERROR_MESSAGE_FIELD, new BString(Constants.SQL_EXCEPTION_OCCURED));
-        } else {
-            sqlConnectorError.put(ERROR_MESSAGE_FIELD, new BString(throwable.getMessage()));
-        }
-        return sqlConnectorError;
+    public static BError getSQLConnectorError(Context context, Throwable throwable) {
+        String detailedErrorMessage = throwable.getMessage() != null ? throwable.getMessage() : Constants.DATABASE_ERROR_MESSAGE;
```
Comment (wggihan) [inline stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java]: Is there any chance to throw any Java specific exception from this throwable?

Comment (Manuri) [reply stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java]: Excepted exceptions are SQLExceptions here which do not have java specific errors. Still I cannot say there is zero probability for a java specific exception to be thrown. 
However, I will be refactoring error handling code to handle only SQL errors/Connection pool errors etc and let the other exceptions be thrown with a custom message.

Comment (wggihan) [reply stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java]: Can you please check the line length here?

**Pull request merged**